Makefile

```diff
@@ -1,13 +1,39 @@
+VERSION         := 0.08
+TARGET          := $(shell uname -r)
+DKMS_ROOT_PATH  := /usr/src/msi_ec-$(VERSION)
```

Comment:
`/usr/src` is a restricted path for the distro and we shouldn't touch it.
We can write safely in `/usr/local/src`

See Linux Filesystem Hierarchy: https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/usr.html

Reply:
Sorry, forget it. DKMS works under `/usr/src` path.